services/generic/door_lock.md

```diff
@@ -1,69 +1,138 @@
+| `lock_type` | `"key"`                    | No       | How lock was activated, it can take values such as `key`, `pin`, `rfid`.                                                                                                                                                                                                                                                                 |
+| `timeout_s` | `"30"`                     | No       | Remaining time in seconds before the lock will be automatically secured again.                                                                                                                                                                                                                                                           |
+| `timeout_m` | `"2"`                      | No       | Remaining time in minutes before the lock will be automatically secured again.                                                                                                                                                                                                                                                           |
+
+## Service properties
+
+| Name                        | Type      | Example                            | Description                                                                                                                                                                                                                                                                    |
+|-----------------------------|-----------|------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `sup_components`            | str_array | `["is_secured", "door_is_closed"]` | List of supported lock configuration components. Possible values are: `is_secured`, `door_is_closed`, `bolt_is_locked`,  `latch_is_closed`.                                                                                                                                    |
+| `sup_op_types`              | str_array | `["timed", "constant"]`            | List of supported operation types. Possible values are: `timed`, `constant`.                                                                                                                                                                                                   |
+| `sup_modes`                 | str_array | `["unsecured", "secured"]`         | List of supported modes. Possible values are: `secured`, `unsecured`, `unsecured_with_timeout`, `unsecured_for_inside_door_handles`, `unsecured_for_inside_door_handles_with_timeout`, `unsecured_for_outside_door_handles`,`unsecured_for_outside_door_handles_with_timeout`. |
```

Comment:
Add note that for timeout options, timeout can be defined in cmd.lock.set_configuration?

Reply:
I think it's already as verbose as needed.